src/DistortableCollection.js

```diff
@@ -199,19 +199,65 @@ L.DistortableCollection = L.FeatureGroup.extend({
     return reduce / imgs.length;
   },
 
-  isJsonDetected(currentURL) {
+  _extractJSONDownloadURL(currentURL) {
```

Comment:
Here I'm thinking... maybe this belongs outside the core lib, since it addresses the URL bar?

Reply:
Okay, but you know it depends on what we agree should be core to the library and what responsibility should be passed to the library user.